### PR TITLE
Add container flow

### PR DESCRIPTION
**Pull request by richlander**
- Add Dockerfile.
- Add README.md with instructions.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+RUN tar -xf wasi-sdk-20.0-linux.tar.gz -C wasi-sdk
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0-preview-jammy
+
+ENV WASI_SDK_PATH=/wasi-sdk/
+COPY --from=installer /wasi-sdk/wasi-sdk-20.0/ $WASI_SDK_PATH
+
+COPY --from=installer /slight/release/slight /usr/bin/
+
+RUN dotnet workload install wasi-experimental -s https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json
```
Comment (SteveSandersonMS) [inline docker/Dockerfile:18]: I *think* the `-s ...` might not be required now preview 4 has shipped, but am not 100% sure.

**Review by SteveSandersonMS — APPROVED** [review]
Nice, thanks!

File: docker/Dockerfile (hunk 1 above)
Comment (richlander) [reply docker/Dockerfile:18]: You are right. I didn't know what your intent was. I'm assuming you want folks to use stable releases now that P4 is shipped. I'll change that.

Comment (SteveSandersonMS) [reply docker/Dockerfile:18]: Oh I probably should have waited for your feedback on this before merging. It's not a big deal but I'm happy to change it further if you think it worthwhile.

Comment (SteveSandersonMS) [reply docker/Dockerfile:18]: OK I did this to save you from having to do another PR: https://github.com/SteveSandersonMS/spiderlightning-dotnet/commit/2cdae45edeae85d3eac571e8c282422715324c90

**Pull request merged**